docs/getting_started/requirements.md

```diff
@@ -15,7 +15,9 @@ Recommendation for production setups is to use Varnish/Fastly, Redis/Memcached,
 
 !!! caution "Recommended versions"
 
-    Using the latest listed version of each product or component is always recommended. Review all the recommended versions carefully. If you see a "+" next to the product version, it means that we recommend this version or higher within the same major release. For example, "Nginx 1.18+" means any 1.x version higher or equal to 1.18, but not 2.x.
+    Review all the recommended versions carefully. If you see a "+" next to the product version, it means that we recommend this version or higher within the same major release. For example, "Nginx 1.18+" means any 1.x version higher or equal to 1.18, but not 2.x.
+
+    Using the latest listed version of each product or component is recommended. Always use a version that receives security updates, either by the vendor themselves or by a trusted third party for example, the distribution vendor.
```

Comment:
Good reshuffle of the text. 👍  There should be a comma after "party". Also I prefer "like" or "such as", to reduce the number of commas and because there aren't many kinds of trusted 3rd parties here.
```suggestion
    Using the latest listed version of each product or component is recommended. Always use a version that receives security updates, either by the vendor themselves or by a trusted third party, such as the distribution vendor.
```